lib/diagnostic.rb

```diff
@@ -108,10 +108,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = {:favorite_number=> 11, :first_name => "Jim"}'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'person.default = "default"'
```

Comment:
You were on the right track here:

`'person.default = "#{person[:first_name]} is great."'`